addons/medical_statemachine/functions/fnc_conditionExecutionDeath.sqf

```diff
@@ -17,4 +17,4 @@
 
 params ["_unit"];
 
-(GVAR(fatalInjuryCondition) < 2) && {!(_unit getVariable [QEGVAR(medical,deathBlocked), false])}
+(isPlayer _unit && {GVAR(fatalInjuriesPlayer) != FATAL_INJURIES_NEVER} || {!isPlayer _unit && {GVAR(fatalInjuriesAI) != FATAL_INJURIES_NEVER}}) && {!(_unit getVariable [QEGVAR(medical,deathBlocked), false])}
```

Comment:
I wouldn't mind seeing this conditional broken across multiple lines. That's just a minor readability thing though.